### PR TITLE
Improve LSP support.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tests/toit/utils.toit [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2024 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import host.pipe
+import system
+
+class ToitBin:
```
Comment (kasperl) [inline tests/toit/utils.toit:8]: I would call this `ToitBinary` or `ToitExecutable`.

File: tests/toit/version-test.toit [hunk 2]
```diff
@@ -6,14 +6,16 @@ import expect show *
...
-  toit-bin := args[0]
-  version := pipe.backticks toit-bin "version"
-  expect-equals "$system.vm-sdk-version$NL" version
+  toit-bin := ToitBin args
+  version := toit-bin.backticks ["version"]
+  expect-equals "$system.vm-sdk-version\n" version
 
-  dash-version := pipe.backticks toit-bin "--version"
-  expect-equals "$system.vm-sdk-version$NL" dash-version
+  // The '--version' is special-cased. We are not allowed to the test-sdk override.
```
Comment (kasperl) [inline tests/toit/version-test.toit:16]: wat - can't grok the last sentence.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tests/toit/utils.toit (hunk 1 above)
Comment (floitsch) [reply tests/toit/utils.toit]: Done.

File: tests/toit/version-test.toit (hunk 2 above)
Comment (floitsch) [reply tests/toit/version-test.toit]: done.

**Pull request merged**